src/local_storage.test.js

```diff
@@ -79,7 +79,7 @@ describe('LocalStorage Populate and validate', () => {
     // Addresses in the AB are ordered by Name, so they might not be in the same order as they are in the localStorage
     addressBookEntries.forEach((address) => {
       const addressIncluded = localStorageAddressBook.includes(address)
-      if (!addressIncluded) console.log('address not found = ', address)
+      if (!addressIncluded) console.info('address not found = ', address)
```

Comment:
@francovenica this was [breaking the script](https://github.com/gnosis/safe-react-e2e-tests/pull/72/files#diff-255c90a739ad573d1971cd0679b7a0cf8d16a1c45f624ffe587dc8f3884cfa63L40-L45) for some reason, so I replaced it with console.info.